al_run_list/get_run_list.py

```diff
@@ -21,11 +21,10 @@ def connectMySQL(run_num):
     '''Read in yaml configuration file and return a connection using the data from the file.'''
 
     #Open connection and read it into directory
-    mysql_db = None
-    with open('qweak_db_%s_read_cred.yml'%(run_num)) as f:
+    with open('qweak_db_{0}_read_cred.yml'.format(run_num)) as f:
         mysql_db = yaml.safe_load(f)
-        if mysql_db == None:
-            print('Cannot open qweak_db_run1_read_cred.yml, is the config correct?')
+        if mysql_db is None:
+            raise Exception('Cannot open qweak_db_run1_read_cred.yml, is the config correct?')
```

Comment:
OK yeah I just double checked, it will return None if there is no information in f.
  - http://pyyaml.org/wiki/PyYAMLDocumentation

Watch your wording too. You _have_ opened the file already. This is `None` if there was no information in the yaml file.